ftw/news/browser/resources/news.scss

```diff
@@ -2,69 +2,89 @@
 @include portal-type-font-awesome-icon(ftw-news-newsfolder, newspaper-o);
 @include portal-type-font-awesome-icon(ftw-news-newslistingblock, newspaper-o);
 
-.portletItemDate {
-  color: $color-text-light;
-  font-size: $font-size-small;
-  margin-bottom: $margin-heading-vertical;
-}
+$leadimage-width: 200px;
```

Comment:
@jone This value depends on the value configured in the `how to handle images` configuration in the plone. So would it be possible to get the width configured for news out of the code.